### DISCUSSION TITLE
Opening the edit URL in a new browser tab first loads the listing page and then subsequently loads the entity.

### DISCUSSION THREAD

**Discussion by EdwinThomas11** [serenity-is/Serenity#6968] [Help] [answered]
Hello Team,

I'm looking to open an edit dialog in a new browser tab from the listing page. Currently, I can achieve this using `window.open()`, but it loads the listing page first and then subsequently loads the entity dialog. I'm seeking assistance to bypass the listing page and load the entity dialog directly in a new browser tab. Your help with this would be greatly appreciated.

@volkanceylan @VictorTomaili Appreciate your inputs here.

Thanks,
Edwin Thomas D

**Comment by dfaruque** [comment, marked as answer]
The following sample might be helpful for your case. it does not load listing page first.
https://demo.serenity.is/BasicSamples/EntityDialogAsPanel/11077

**Reply by EdwinThomas11** [reply]
Thank you for your response; it's greatly appreciated.
